var/spack/repos/builtin/packages/vecgeom/package.py

```diff
@@ -20,10 +20,16 @@ class Vecgeom(CMakePackage, CudaPackage):
     maintainers("drbenmorgan", "sethrj")
 
     version("master", branch="master")
+    version(
+        "2.0.0-surfacedev",
+        url="https://gitlab.cern.ch/-/project/135666/uploads/47afee17a8566c225e5d57df620c2800/VecGeom-v2.0.0-surfacedev.tar.gz",
```

Comment:
Can we add a comment with a link to the fork this is on? https://gitlab.cern.ch/-/project/135666 gets me a CERN login.